<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="user">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_7">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>430</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Acumin Pro</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Numéro Salle</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="numSalle">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>430</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="etat">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>470</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>113</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>470</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Acumin Pro</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Etat Salle</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouterS">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#ajouterS
{
background-color:transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="afficherS">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#afficherS
{
background-color:transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimerS">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#supprimerS
{
background-color:transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_2
{
background-color:transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>401</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton
{
background-color:transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="bg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update1">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>490</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#update1
{
background-color:transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="update">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>490</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#update{
background-color:transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>bg</zorder>
     <zorder>label_5</zorder>
     <zorder>numSalle</zorder>
     <zorder>etat</zorder>
     <zorder>label_6</zorder>
     <zorder>ajouterS</zorder>
     <zorder>afficherS</zorder>
     <zorder>supprimerS</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>tableView_2</zorder>
     <zorder>pushButton</zorder>
     <zorder>update1</zorder>
     <zorder>update</zorder>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>480</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Acumin Pro</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Numéro Salle</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#modifier
{
background-color:transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="afficher">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>172</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#afficher
{
background-color:transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>480</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Acumin Pro</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Heure</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#ajouter
{
background-color:transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>450</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Acumin Pro</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Numéro Réservation</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="heure">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>480</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#supprimer
{
background-color:transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="numS">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>480</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>450</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Acumin Pro</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>130</y>
        <width>521</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="num">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>450</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="date">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>450</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherche">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>90</y>
        <width>421</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="bg1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_3
{
background-color:transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Acumin Pro</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <zorder>bg1</zorder>
     <zorder>label_4</zorder>
     <zorder>modifier</zorder>
     <zorder>afficher</zorder>
     <zorder>label_3</zorder>
     <zorder>ajouter</zorder>
     <zorder>label</zorder>
     <zorder>heure</zorder>
     <zorder>supprimer</zorder>
     <zorder>numS</zorder>
     <zorder>label_2</zorder>
     <zorder>tableView</zorder>
     <zorder>num</zorder>
     <zorder>date</zorder>
     <zorder>recherche</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>label_7</zorder>
    </widget>
   </widget>
   <widget class="QPushButton" name="admin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Administrateur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enseignant">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enseignant</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
